### PR TITLE
remove unnecessary documentation deps from the actual package

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 51ce44f8 is formatted properly.

File: docs/Project.toml [hunk 1]
```diff
@@ -12,3 +12,4 @@ Tulip = "6dd1b50a-3aae-11e9-10b5-ef983d2400fa"
 
 [compat]
 Documenter = "0.26"
+Literate = "0.28"
```
Comment (laurentheirendt) [inline docs/Project.toml:15]: before the version was `2.8`, now `0.28`. is this correct?

Comment (exaexa) [reply docs/Project.toml:15]: no it isn't and I've got no idea how it could get here. :fearful:

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 2b2a84b8 is formatted properly.

**Review by laurentheirendt — APPROVED** [review]

**Pull request merged**